R/prep_access.R

```diff
@@ -0,0 +1,40 @@
+#' Prepare access
+#'
+#' Extract variableNames for a given data file and add them to the attributes.csv
+#' @param data_path path to the data folder. Defaults to "data" and R 'data' file types
+#' @param access_path path to the access.csv file. Defaults to "data/metadata/access.csv".
+#'
+#' @return the functions writes out the updated access.csv file to access_path.
+#' @export
+prep_access <- function(data_path = here::here("data"),
+                        access_path = here::here("data", "metadata",
+                                                 "access.csv")
+                        ){
+
+  if(!file.exists(data_path)){stop("invalid path to data folder")}
+  if(!file.exists(access_path)){
+    stop("access file does not exist. Check path or run create_spice?")}
+
+  access <- readr::read_csv(access_path, col_types = readr::cols())
```

Comment:
is the `col_types = readr::cols()` strictly necessary here? I think setting this arg to `readr::cols()` has the same effect as setting it to `NULL` which is the default.